sci-physics/geant/files/geant-4.11.0.2-musl-avoid-execinfo.patch

```diff
@@ -0,0 +1,61 @@
+# Avoid including execinfo and calling backtrace function when execinfo.h is
+# not present or supplied by system's libc
+#
+# Closes: https://bugs.gentoo.org/829151
+--- a/source/global/management/include/G4Backtrace.hh
++++ b/source/global/management/include/G4Backtrace.hh
+@@ -87,7 +87,9 @@
+
+ #if defined(G4UNIX) && !defined(WIN32)
+ #  include <cxxabi.h>
+-#  include <execinfo.h>
++#  if defined(G4DEMANGLE_AVAILABLE)
++#     include <execinfo.h>
++#  endif
+ #  include <unistd.h>
+ #endif
+
+@@ -109,7 +111,9 @@ using G4ResultOf_t = std::invoke_result_t<FuncT, ArgTypes...>;
+ #    define G4SIGNAL_AVAILABLE
+ #  endif
+ #  if !defined(G4DEMANGLE_AVAILABLE)
+-#    define G4DEMANGLE_AVAILABLE
++#    if defined(G4UNIX) && defined(HAVE_EXECINFO_H)
++#       define G4DEMANGLE_AVAILABLE
```

Comment:
This needs to come before `if defined(G4DEMANGLE_AVAILABLE)` (on line 12 above), right?

Reply:
Yep, mu bad, will fix

Reply:
@amadio how about now?

Reply:
I think that now it looks alright. I'd also like to submit this as a patch upstream if you don't mind. For commit author/email, can I use the same as for other commits you've done in Gentoo? Please let me know.

Reply:
Yeah, sure.